src/frontend/js/components/SearchFilterGroupModal/_styles.scss

```diff
@@ -70,6 +70,7 @@
     background: r-theme-val(search-filters-group-modal, close-background);
     border: none;
     border-top: 1px solid r-theme-val(search-filters-group-modal, close-border);
+    margin-top: -1px; // Avoid having a double border when the button overlaps with a list item
```

Comment:
What do you think to not apply a border-bottom to the latest button item?
```css
&__item {
  ....
  &:not(:last-child) > button {
    border-bottom: 1px solid r-theme-val(search-filters-group-modal, item-border);
  }
}
```

Reply:
With the fixed-height modal this is not ideal as it leaves the last element without a border if the list it not long enough to reach the bottom of the modal.